### DISCUSSION TITLE
[Feature proposal] Add `scroll-behavior` support

### DISCUSSION THREAD

**Discussion by dmitry-ivanov** [tailwindlabs/tailwindcss#5007] [Ideas]
Hi guys,

A relatively new CSS option introduces smooth scrolling by adding just one line of CSS - [`scroll-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior).

It was already mentioned [here](https://github.com/tailwindlabs/tailwindcss/discussions/1406#discussioncomment-56011), but I think it's worth its own discussion.

It would be handy to have it supported in TailwindCSS, because it allows to achieve smooth scrolling without using JavaScript.

Thank you!
Keep doing a great job! 👍

**Comment by lukewarlow** [comment]
This would certainly be a useful addition for creating carousels (along with scroll-snap support)

In the meantime I've made https://github.com/lukewarlow/tailwind-scroll-behavior to help me with this.

**Reply by lukewarlow** [reply]
I've made a PR to add this to Tailwind: https://github.com/tailwindlabs/tailwindcss/pull/5388. Hopefully it makes it into v3.

**Reply by lukewarlow** [reply]
This PR has now been merged
